pkg/containerruntime/containerruntime.go

```diff
@@ -130,6 +137,7 @@ func (cfg Config) Engine(kubeletVersion *semver.Version) Engine {
 		registryMirrors:     cfg.RegistryMirrors,
 		sandboxImage:        cfg.SandboxImage,
 		registryCredentials: cfg.RegistryCredentials,
+		version:             cfg.ContainerdVersion,
```

Comment:
I don't see that we use this version anywhere. I think that we might need to add it in two places:

https://github.com/kubermatic/machine-controller/blob/e7e4ba8eb535086374108ed235c2e33bab7f4292/pkg/containerruntime/containerd.go#L67

https://github.com/kubermatic/machine-controller/blob/e7e4ba8eb535086374108ed235c2e33bab7f4292/pkg/containerruntime/docker.go#L88

Reply:
https://github.com/kubermatic/machine-controller/blob/e7e4ba8eb535086374108ed235c2e33bab7f4292/pkg/containerruntime/containerd.go#L67 gets overwritten by the value passed via CLI (if present) at https://github.com/kubermatic/machine-controller/blob/e7e4ba8eb535086374108ed235c2e33bab7f4292/pkg/containerruntime/containerd.go#L71

As for the docker runtime: I added the same mechanism already in place for containerd.

WDYT?

Reply:
This looks good now!